apps/api_web/config/config.exs

```diff
@@ -62,6 +62,11 @@ config :mime, :types, %{
 
 config :ja_serializer, key_format: :underscored
 
+config :recaptcha,
+  verify_url: "https://www.google.com/recaptcha/api/siteverify",
+  public_key: {:system, "RECAPTCHA_PUBLIC_KEY"},
+  secret: {:system, "RECAPTCHA_PRIVATE_KEY"}
```

Comment:
suggestion: for dynamic configuration, we typically load it using [`System.get_env/2` ](https://hexdocs.pm/elixir/1.14.3/System.html#get_env/2)in `config/runtime.exs` now.

Reply:
> suggestion: for dynamic configuration, we typically load it using [`System.get_env/2` ](https://hexdocs.pm/elixir/1.14.3/System.html#get_env/2)in `config/runtime.exs` now.

I added the env vars to that file, and added a feature flag to disable the recaptcha code outside of prod builds.

Reply:
> I tried it out, and the first thing I noticed was that the form fields were cleared if I submitted without the captcha. Otherwise it seems to work well.

I added some frontend code to only enable the submit button once the user completes the recaptcha, so this won't happen outside of someone attempting to do something weird. 

Deployed to dev-green 🙂

Reply:
> I tried it out, and the first thing I noticed was that the form fields were cleared if I submitted without the captcha. Otherwise it seems to work well.

I added some frontend code to only enable the submit button once the user completes the recaptcha, so this won't happen outside of someone attempting to do something weird. 

Deployed to dev-green 🙂